### ISSUE TITLE
[Help] How to JsonSerializableConverter by type of response

### ISSUE THREAD

**Issue by kimthangatm** [lejard-h/chopper#31]
I have two response (single and array like)

```json
{
  "type": "topic",
  "data": {
    "_id": "5c6d2bccf0059484c041993c",
    "name": "Demo topic 1"
  }
}
```

```json
{
  "type": "topics",
  "data": [
    {
      "_id": "5c6d2bccf0059484c041993c",
      "name": "Demo topic 1"
    },
    {
      "_id": "5c6d5bec115be334f18f8c48",
      "name": "Demo topic 2"
    }
  ],
  "meta": {
    "total": 200
  }
}
```

**Comment by zgunz42** [comment]
Add a ResponseInterceptor to convert data on single result to array based on type

**Comment by lejard-h** [comment]
Do you have more infos about what you are trying to do ?

To me it does not seem to be related to Chopper. How did you build your JSON ? straight from a JsonSerializable serializer ?

Did you see this exemple ? https://github.com/lejard-h/chopper/blob/master/example/bin/main_json_serializable.dart#L60

**Comment by MichaelDark** [comment]
@kimthangatm I think this issue is closed by #34 
Have a look at 2.2.0 and give a feedback
